src/components/Albums/Albums.js

```diff
@@ -84,13 +24,27 @@ export class AlbumsComponent {
     });
   }
 
+  fetchAlbumsCollectionData() {
+    this.state.isFetching = true;
+    MusicService.getAlbums().then(albums => {
+      this.state.albums = albums.map(album =>
+        Object.assign(album, { authors: album.authors.join(", ") })
+      );
+      this.state.isFetching = false;
+      this.mountPoint.innerHTML = this.render();
+      this.querySelectors();
+      this.initMaterial();
+    });
+  }
+
   mount() {
```

Comment:
could you please add parameter to `mount(shouldFetchData = true)`? If `shouldFetchData` is true - then you will call `fetchAlbumsCollectionData `. By default it should be true.

This way you will be able to reuse this method inside `fetchAlbumsCollectionData ` method

Reply:
could you please explain in more detail.

Reply:
Done